src/atoms/Button/Button.tsx

```diff
@@ -26,11 +28,18 @@ export const Button = styled(Typography)<Props>`
 
   &:focus,
   &:hover {
-    background: ${props => props.theme.primaryDark};
+    background: ${props =>
+      props.secondary ? props.theme.primary : props.theme.primaryDark};
```

Comment:
In Storybook it seems to still be using the dark color (#006077) in the condary variant, though I'm not sure why

Reply:
![image](https://user-images.githubusercontent.com/434238/48509023-02e49d80-e81e-11e8-96dd-96a83773f013.png)
 It seems to be behaving properly on my end

Reply:
idk what happened, but it seems to be fixed now.